build_locally.sh

```diff
@@ -1,13 +1,10 @@
 #!/bin/sh
 
-# Install rosdoc_lite if it isn't there yet
-test -x `which rosdoc_lite` || sudo apt install ros-$ROS_DISTRO-rosdoc-lite
```

Comment:
@MarqRazz i think we should keep this logic, except:

1) have it install sphinx rather than rosdoc_lite (sphinx was a dependency of rosdoc_lite)
2) update the conditional logic to actually work. i did this yesterday for the moveit1 tutorials [here](https://github.com/ros-planning/moveit_tutorials/blob/master/build_locally.sh#L4), just copy that logic

Reply:
can you open a new PR for this?